tests/test_ssl.py

```diff
@@ -0,0 +1,25 @@
+# import asyncio
+# import ssl
+#
+# from aiomysql import create_pool
+#
+#
+# async def main():
+#     ctx = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
```

Comment:
I think the latest protocol should be used which I think is PROTOCOL_TLSv1

Reply:
Yeah I completely agree though vaguely remember when I tested it that it didn't like PROTOCOL_TLSv1, but its possible that it was the MySQL i was testing against instead of RDS.